unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/DCLVideoTexture.cs

```diff
@@ -312,11 +323,14 @@ private bool IsPlayerInSameSceneAsComponent(string currentSceneId)
         }
 
         private void OnPlayerCoordsChanged(Vector2Int coords, Vector2Int prevCoords)
-        {
+        {   
             SetPlayStateDirty();
         }
 
-        private void OnSceneIDChanged(string current, string previous) { isPlayerInScene = IsPlayerInSameSceneAsComponent(current); }
+        private void OnSceneIDChanged(string current, string previous) 
+        { 
+            isPlayerInScene = IsPlayerInSameSceneAsComponent(current);
+        }
```

Comment:
What about using lambda expression here (to align with our code-style)?
```suggestion
        private void OnSceneIDChanged(string current, string previous) =>
            isPlayerInScene = IsPlayerInSameSceneAsComponent(current);
```

Reply:
done ;)